### DISCUSSION TITLE
Migrating from 'arr-SMA to -extended

### DISCUSSION THREAD

**Discussion by tunes0710** [RandomNinjaAtk/docker-radarr-extended#19] [Q&A] [answered]
Hiya,
I am running your *-SMA dockers on unraid, but notice they are showing broken links now (where your project has been archived).
Is there any way to switch to the extended one whilst retaining all my settings/configs?
Is it a case in my current radarr docker settings, changing the repository from randomninjaatk/radarr-sma to something else?
Cheers :)

**Comment by RandomNinjaAtk** [comment, marked as answer]
Probably the easiest thing to do would be to remove the current docker, pull the new docker via the unraid app store template, and adjust the default config directory to your previous installation. 

The other method would be changing the docker image source to the new location. But you would be missing the latest settings and would have to manually update those. Where as the process above should give you all the latest settings.

One thing to keep in mind though, if you were on nightly release, if the current develop release is not newer than your nightly installation, it may not work because of version differences. So you should check your installed version number vs the current develop version number. It also would be a good idea to take a backup of the data, incase things don't go as expected.
